cli/command/system/inspect.go

```diff
@@ -105,30 +105,51 @@ func inspectAll(ctx context.Context, dockerCli *command.DockerCli, getSize bool,
 	var inspectAutodetect = []struct {
 		ObjectType      string
 		IsSizeSupported bool
+		IsSwarmObject   bool
```

Comment:
Non-blocking observation: There's no reason for the fields in this structure to be exported.